### ISSUE TITLE
form-control class is everywhere

### ISSUE THREAD

**Issue by Mikecoon** [unclead/yii2-multiple-input#198]
In BaseColumn you have hardcoded **form-control** class for every field, and there is no way to remove it.

```
protected function renderDefault($name, $value, $options)
    {
        $type = $this->type;

        if (method_exists('yii\helpers\Html', $type)) {
            Html::addCssClass($options, 'form-control');
            $input = Html::$type($name, $value, $options);
        } elseif (class_exists($type) && method_exists($type, 'widget')) {
            $input = $this->renderWidget($type, $name, $value, $options);
        } else {
            throw new InvalidConfigException("Invalid column type '$type'");
        }

        return $input;
    }
```

Why? It painful to use it without default bootstrap theme. For example, I got this extra block.

![image](https://user-images.githubusercontent.com/5504685/36136582-50b831c4-10a2-11e8-961c-6fe92893974a.png)

My suggest is to make it optionable. 
```
if (!isset($options['class']))
     Html::addCssClass($options, 'form-control');
```

**Comment by unclead** [comment]
i agree that we can make it configurable but by default better to leave it as is for BC. Can you make a PR?

**Comment by unclead** [comment]
@Mikecoon so, will you make a PR? 😄

**Comment by Mikecoon** [comment]
Sorry, little busy these days. :(

**Comment by unclead** [comment]
fixed via 46bb2af5b801ff20b673a837b0c8825502343e2b in which new option `theme` was introduced. I left bootstrap theme by default to keep BC but I think we will be able to switch to default theme in next major release
